using roboscape after listening to robots crashes the server

```
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox::trace received "record-action" message from hamid (_netsblox1538765850188)
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:storage:user-actions:trace about to store event from session: _netsblox1538765850188/1538765825093
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox::trace received "user-action" message from hamid (_netsblox1538765850188)
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox::log rejecting action with id 43 (104) from _netsblox1538765850188 at myRole-1534948470517 in 5b7d74e97b167616f3949564
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox::trace Sending message to _netsblox1538765850188 "{"type":"action-rejected","action":{"isUserAction":true,"type":"startScript","args":["item_28",true],"user":"_netsblox1538765850188","username":"hamid","id":43,"time":1538768737837}}"
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:storage:user-actions:trace successfully recorded event from session _netsblox1538765850188/1538765825093
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:middleware:trace validating cookie
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:services:info Received request to RoboScape for listen (from _netsblox1538765850188)
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:services:log calling RoboScape.listen(["66fc"]) _netsblox1538765850188
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:services:roboscape:log unregister _netsblox1538765850188 00409d8f66fc
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:04 GMT netsblox:services:roboscape:log register _netsblox1538765850188 00409d8f66fc
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox::trace received "record-action" message from hamid (_netsblox1538765850188)
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:storage:user-actions:trace about to store event from session: _netsblox1538765850188/1538765825093
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox::trace received "user-action" message from hamid (_netsblox1538765850188)
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox::log rejecting action with id 43 (104) from _netsblox1538765850188 at myRole-1534948470517 in 5b7d74e97b167616f3949564
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox::trace Sending message to _netsblox1538765850188 "{"type":"action-rejected","action":{"isUserAction":true,"type":"startScript","args":["item_0",true],"user":"_netsblox1538765850188","username":"hamid","id":43,"time":1538768742949}}"
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:storage:user-actions:trace successfully recorded event from session _netsblox1538765850188/1538765825093
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:middleware:trace validating cookie
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:services:info Received request to RoboScape for send (from _netsblox1538765850188)
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:services:log calling RoboScape.send("66fc","beep 100 100") _netsblox1538765850188
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox::trace Sending message to _netsblox1538765850188 "{"msgType":"robot command","content":{"robot":"00409d8f66fc","command":"beep 100 100"},"type":"message","dstId":"everyone in room"}"
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:services:roboscape:log set beep 00409d8f66fc 100 100
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:09 GMT netsblox:services:roboscape:log event beep {"msec":100,"tone":100,"robot":"00409d8f66fc","time":3005347}
6|netsblox-dev  | [ERROR] ReferenceError: content is not defined
6|netsblox-dev  |     at /home/ubuntu/netsblox-dev/src/server/rpc/procedures/roboscape/roboscape.js:337:35
6|netsblox-dev  |     at Array.forEach (<anonymous>)
6|netsblox-dev  |     at Robot.sendToClient (/home/ubuntu/netsblox-dev/src/server/rpc/procedures/roboscape/roboscape.js:326:18)
6|netsblox-dev  |     at Robot.onMessage (/home/ubuntu/netsblox-dev/src/server/rpc/procedures/roboscape/roboscape.js:372:14)
6|netsblox-dev  |     at Socket.<anonymous> (/home/ubuntu/netsblox-dev/src/server/rpc/procedures/roboscape/roboscape.js:918:15)
6|netsblox-dev  |     at emitTwo (events.js:126:13)
6|netsblox-dev  |     at Socket.emit (events.js:214:7)
6|netsblox-dev  |     at UDP.onMessage [as onmessage] (dgram.js:659:8)
6|netsblox-dev  | 
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:blob:s3:trace Connecting to s3.amazonaws.com
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:blob:s3:trace Using access key id: "AKIAISTYNTKQZLV6IVLQ"
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:bug-reporter:trace sending bug reports to http://netsblox.tk:9753 and dellydela@gmail.com
6|netsblox-dev  | ROBOSCAPE_PORT is 1973
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:jsdoc:warn can't associate /**, * Get the maximum longitude of the current map., * @returns {Longitude}, */ with any function. # Fix it at line 343, column 4
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:jsdoc:warn can't associate /**, * Get the maximum latitude of the current map., * @returns {Longitude}, */ with any function. # Fix it at line 349, column 4
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:jsdoc:warn can't associate /**, * Get the minimum longitude of the current map., * @returns {Longitude}, */ with any function. # Fix it at line 355, column 4
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:jsdoc:warn can't associate /**, * NPlayer - This constructor is called on the first request to an RPC, * from a given room, *, * @constructor, * @return {undefined}, */ with any function. # Fix it at line 9, column 4
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for AirQuality
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Battleship
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for BritishMuseum
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Chart
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for CloudVariables
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for ConnectN
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Dev
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Earthquakes
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Eclipse2017
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Execute
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for Geolocation
6|netsblox-dev  | Fri, 05 Oct 2018 19:46:10 GMT netsblox:services:info Adding route for GoogleMaps
```

--- a/src/server/rpc/procedures/roboscape/roboscape.js
+++ b/src/server/rpc/procedures/roboscape/roboscape.js
@@ -337,11 +337,11 @@ Robot.prototype.sendToClient = function (msgType, content, fields) {
                     text += ' ' + content[fields[i]];
                 }
 
-                const content = {
+                const encryptedContent = {
                     robot: myself.mac_addr,
                     message: myself.encrypt(text.trim())
                 };
-                socket.sendMessage('robot message', content);
+                socket.sendMessage('robot message', encryptedContent);
             }
         } else {
             logger.log('socket not found for ' + uuid);

